app/src/main/java/com/strv/movies/ui/moviedetail/MovieDetailScreen.kt

```diff
@@ -82,29 +84,49 @@ fun MovieTrailerPlayer(videoId: String, progressSeconds: MutableState<Float>) {
 
 @Composable
 fun MoviePoster(movie: MovieDetail) {
-    AsyncImage(
-        model = "https://image.tmdb.org/t/p/w500${movie.posterPath}",
-        contentDescription = stringResource(id = R.string.movie_image),
-        modifier = Modifier
-            .padding(top = 16.dp)
-            .size(120.dp)
-    )
+    Column {
+
+        AsyncImage(
+            model = "https://image.tmdb.org/t/p/w500${movie.posterPath}",
+            contentDescription = stringResource(id = R.string.movie_image),
+            modifier = Modifier
+                .padding(top = 16.dp)
+                .size(120.dp),
+        )
+        Row {
+            Text(
+                movie.title,
+                fontWeight = FontWeight.Bold,
+                textAlign = TextAlign.Center,
+                fontSize = 10.sp,
+                color = Color.Red
+            )
+        }
+    }
 }
 
 @Composable
 fun MovieInfo(movie: MovieDetail) {
     Column {
         Text(
             movie.title,
-            modifier = Modifier.padding(top = 16.dp, end = 16.dp),
+            modifier = Modifier
+                .padding(start = 10.dp, top = 16.dp, end = 10.dp),
             fontWeight = FontWeight.Bold,
             fontSize = 20.sp
         )
-        Text(movie.releaseDate, modifier = Modifier.padding(top = 8.dp))
+        Text(
+            movie.releaseDate,
+            modifier = Modifier
+                .padding(start = 10.dp, top = 8.dp)
+        )
         Text(
             movie.overview,
-            modifier = Modifier.padding(top = 8.dp, end = 16.dp),
-            textAlign = TextAlign.Justify
+            modifier = Modifier
+                .padding(start = 10.dp, top = 8.dp, end = 10.dp)
+                .size(240.dp),
```

Comment:
When you using static size for Text element there can be situation in which contect of this element do not fit in it. In this situation it is good to use overflow: 
`overflow = TextOverflow.Ellipsis`
It will add "..." to the end of the text to notify user that there is more of it.

Reply:
Got it with ` maxLines=1,
 overflow = TextOverflow.Ellipsis,`